problems/1.types+operators.js

```diff
@@ -2,16 +2,26 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  if (val !== NaN) {
```

Comment:
We don't care if val is !== NaN we care that it's type is equal to "number". Fun fact NaN !== NaN evaluates to true.